### PR TITLE
aspnet

### PR DESCRIPTION
**Pull request by bruno-garcia**
Point out common issues (mainly proxy and tls)

### CODE REVIEW

**Review by dcramer — COMMENTED** [review]
remove the redirect_from i think?

File: src/platforms/dotnet/guides/aspnet/index.mdx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+redirect_from:
+  - /platforms/dotnet/aspnet/
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/aspnet/index.mdx:3]:
```suggestion
```

**Comment by bruno-garcia** [comment]
@dcramer good catch. Thanks.

**Review by dcramer — APPROVED** [review]
sdk will auto inherit from dotnet fwiw, so you dont need to define if doesnt have an aspnet specific one

File: src/platforms/dotnet/guides/aspnet/index.mdx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    </defaultProxy>
+</system.net>
+```
+
+### TLS 1.2 Support and Windows Server
+
+If you're not able to capture events from ASP.NET to Sentry,
+on older versions of Windows Server, with older .NET Framework you must enable TLS 1.2.
+
+`ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12`
```
Comment (dcramer) [inline src/platforms/dotnet/guides/aspnet/index.mdx:106]:
````suggestion
```
ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12
```
````

File: src/platforms/dotnet/guides/aspnet/index.mdx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+- An [sample with ASP.NET and EF 6](https://github.com/getsentry/sentry-dotnet-samples/tree/master/AspNetMvc5Ef6).
```
Comment (dcramer) [inline src/platforms/dotnet/guides/aspnet/index.mdx:108]:
```suggestion
- A [sample with ASP.NET and EF 6](https://github.com/getsentry/sentry-dotnet-samples/tree/master/AspNetMvc5Ef6).
```

File: src/platforms/dotnet/guides/aspnet/index.mdx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+- For [more samples](https://github.com/getsentry/sentry-dotnet-samples) of the .NET SDKs
```
Comment (dcramer) [inline src/platforms/dotnet/guides/aspnet/index.mdx:109]:
```suggestion
- [More samples](https://github.com/getsentry/sentry-dotnet-samples) of the .NET SDKs
```

**Pull request merged**